scripts/ai/FillLevelManager.lua

```diff
@@ -154,30 +155,32 @@ end
 ------------------------------------------------------------------------------------------------------------------------
 --- Fill Levels
 ---------------------------------------------------------------------------------------------------------------------------
-function FillLevelManager:getAllFillLevels(object, fillLevelInfo)
+function FillLevelManager.getAllFillLevels(object, fillLevelInfo)
     -- get own fill levels
     if object.getFillUnits then
-        for _, fillUnit in pairs(object:getFillUnits()) do
-            local fillType = self:getFillTypeFromFillUnit(fillUnit)
-            local fillTypeName = g_fillTypeManager:getFillTypeNameByIndex(fillType)
-            self:debugSparse('%s: Fill levels: %s: %.1f/%.1f', object:getName(), fillTypeName, fillUnit.fillLevel, fillUnit.capacity)
-            if not fillLevelInfo[fillType] then fillLevelInfo[fillType] = {fillLevel=0, capacity=0} end
-            fillLevelInfo[fillType].fillLevel = fillLevelInfo[fillType].fillLevel + fillUnit.fillLevel
-            fillLevelInfo[fillType].capacity = fillLevelInfo[fillType].capacity + fillUnit.capacity
-            --used to check treePlanter fillLevel
-            local treePlanterSpec = object.spec_treePlanter
-            if treePlanterSpec then
-                fillLevelInfo[fillType].treePlanterSpec = object.spec_treePlanter
+        for index, fillUnit in pairs(object:getFillUnits()) do

```

Comment:
_ in Lua (and in Pyhton) is used for throw away variables we don't care about. No need to put 'index' here, decent IDEs will only complain that it is unused.